src/Pages/Videodisplay.js

```diff
@@ -0,0 +1,172 @@
+import React,{useEffect,useState,Fragment} from 'react'
+import {useParams,Link} from 'react-router-dom'
+import axios from 'axios'
+
+// @desc importing Components
+import {Videocard,PlaylistAction} from '../Comonents'
+
+// importing hooks 
+import {useLike,useSaved,usePlaylist,useAuth} from '../Context'
+
+// @desc common assets
+import playlist from '../Common-Assets/Playlist.svg'
+
+import {getToken} from '../Token'
+
+
+export function Videodisplay() {
+    const {likedDispatch} = useLike()
+    const {savedDispatch} = useSaved()
+    const{PlaylistDispatcher}=usePlaylist()
+    const {loggedIn} = useAuth()
+    const {videoId} = useParams()
+
+    const [FoundVideo,setFoundVideo] = useState()
+    const [liked,setLiked] = useState("")
+    const [saveToggle,setSaveToggle] = useState("")
+    const [notes,setNotes] = useState("")
+    const [OpenModal,setOpenModal] = useState(false)
+    const [showPlaylist,setShowPlaylist] = useState(false)
+    const[loginModal,setLoginModal] = useState(false)
+
+    const token = getToken() !==null ? getToken() : {token:null}
+    
+
+    useEffect(()=>{
+        (async function fetchData(){
+
+        // Finding Video From URL Param
+        const response_videolist = await axios.get(`https://KnightStream.ishanjirety.repl.co/api/videolist?token=${token.token}`)
```

Comment:
Wrap this network calls inside `try...catch` blocks. And also you can move these to the `services` folder and write all network calls there.
This applies to multiple similar instances in the same file